app/views/return-requirements/view.njk

```diff
@@ -0,0 +1,194 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "macros/charge-version-status-tag.njk" import statusTag %}
+
+{% macro newLine(array) %}
+  {% for item in array %}
+    <p class="govuk-!-margin-bottom-1"> {{ item }}</p>
+  {% endfor %}
+{% endmacro %}
+
+{% block breadcrumbs %}
+  {# Back link #}
+  {{ govukBackLink({
+    text: 'Back',
+    href: '/system/licences/' + licenceId + '/set-up'
+  }) }}
+{% endblock %}
+
+{% block content %}
+  {# Main heading #}
+  <div class="govuk-body">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">
+      <span class="govuk-caption-l">Licence {{ licenceRef }}</span>
+      {{ pageTitle }}
+    </h1>
+    {{ statusTag(status) }}
+  </div>
+
+  <div class="govuk-!-margin-bottom-9">
+    <hr class="govuk-section-break  govuk-!-margin-bottom-2 govuk-section-break--visible">
+    {{ govukSummaryList({
+      classes: 'govuk-!-margin-bottom-2',
+      rows: [
+        {
+          classes: 'govuk-summary-list govuk-summary-list__row--no-border',
+          key: {
+            text: "Start date"
+          },
+          value: {
+            html: '<span data-test="start-date">' + startDate + '</span>'
+          }
+        },
+        {
+          classes: 'govuk-summary-list govuk-summary-list__row--no-border',
+          key: {
+          text: "Reason"
+        },
+          value: {
+          html: '<span data-test="reason">' + reason + '</span>'
+        }
+        }
+      ]
+    }) }}
+    <hr class="govuk-section-break  govuk-!-margin-bottom-2 govuk-section-break--visible">
+  </div>
+
+{% if notes %}
+  <div class="govuk-!-margin-bottom-9">
+    <h2 class="govuk-heading-l govuk-!-margin-bottom-4">Notes</h2>
+    <hr class="govuk-section-break  govuk-!-margin-bottom-2 govuk-section-break--visible">
+    {{ govukSummaryList({
+      classes: 'govuk-!-margin-bottom-2',
+      rows: [
```

Comment:
Currently awaiting a decision on the user email. 

Currently this is not stored nor does it exists in the legacy data. As this affect many records a redesign might be required.